PHP-Math-Functions-max()-#2063 (#2083)
Codecademy/docs: content/php/concepts/math-functions/terms/abs/max/max.md

Reviewer: This file needs to be moved from `content/php/concepts/math-functions/terms/abs/max/max.md` to `content/php/concepts/math-functions/terms/max/max.md`

---
Title: 'Max()'

> Description: 'The max() function in PHP is used to find the highest value in an array or a list of variables. It returns the highest value from the input provided.'

Reviewer: ```suggestion
Description: 'Used to find the highest value in an array or a list of variables. It returns the highest value from the input provided.'
```

Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
Tags:
 - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

The **`max()`** function in PHP is used to find the highest value in an [array](https://www.codecademy.com/resources/docs/php/arrays) or a list of [variables](https://www.codecademy.com/resources/docs/php/variables). It returns the highest value from the input provided.

## Syntax

```pseudo
max($var1, $var2, $var3, ...);
```

> $var1, $var2, $var3: These are the [variables](https://www.codecademy.com/resources/docs/php/variables) or [array](https://www.codecademy.com/resources/docs/php/arrays) from which the maximum value will be returned.

Reviewer: ```suggestion
`$var1`, `$var2`, `$var3` are the [variables](https://www.codecademy.com/resources/docs/php/variables) or [array](https://www.codecademy.com/resources/docs/php/arrays) from which the maximum value will be returned.
```

This still needs to distinguish between calling with multiple variables and calling with a single array.

## Example

The example uses the `max()` function to find the highest value in the array of numbers and then prints the result using the `echo` statement. The output will be the maximum value found in the array.

```php
$numbers = array(23, 12, 35, 67, 8, 90);
$max_value = max($numbers);
echo "The maximum value in the array is: " . $max_value;
```

> ```shell

Reviewer: ```suggestion
This example results in the following output.

```shell
```

> The example uses the `max()` function to find the highest value in the array of numbers and then prints the result using the `echo` statement. The output will be the maximum value found in the array.

Reviewer: Move this sentence to before the example code block.

---
Title: 'Max()'
Description: 'Used to find the highest value in an array or a list of variables. It returns the highest value from the input provided.'
Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
Tags:
 - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

The **`max()`** function in PHP is used to find the highest value in an [array](https://www.codecademy.com/resources/docs/php/arrays) or a list of [variables](https://www.codecademy.com/resources/docs/php/variables). It returns the highest value from the input provided.

## Syntax

```pseudo
max($var1, $var2, $var3, ...);
```

> `$var1`, `$var2`, `$var3` are the [variables](https://www.codecademy.com/resources/docs/php/variables) or [array](https://www.codecademy.com/resources/docs/php/arrays) from which the maximum value will be returned.

Reviewer: Need to mention the difference between calling this with a list of variables (multiple parameters) and calling it with an array (single parameter)
